services/frontend/src/components/Panels/WarningsInfo/WarningsInfo.tsx

```diff
@@ -0,0 +1,234 @@
+  const { name } = useParams()
+  const { setUIOption } = useUIOptions()
+
+  const dispatchTrigger = (isOpen: boolean) =>
+    setUIOption({
+      name: consts.WARNING_WINDOW_SP_IS_ACTIVE,
+      value: isOpen,
+    })
+
+  useEffect(() => {
+    dispatchTrigger(true)
+    return () => dispatchTrigger(false)
+  }, [])
+
+  const initState = privateDists?.data?.sort(
+    (a: BotInfoInterface, b: BotInfoInterface) => a.id - b.id
+  )
+
+  const data = initState?.map((el: BotInfoInterface) => {
+    const components = getAllComponents(el.name || '')
+    const result = components.data?.skills
+      ?.filter(el => el.name !== 'dummy_skill')
+      .map((el: ISkill) => {
+        const resultExamination = examination(el)
+        return { name: el.display_name, data: resultExamination, skill: el }
+      })
+
+    return { name: el.display_name, skill: result, bot: el }
+  })
+
+  const renderMessage = (key: string, data: ICollectionError) => {
+    if (data[key].length === 0) {
+      return null
+    }
+    const message = data[key].map((el: string, i: number) => {
+      return (
+        <div
+          key={i}
+          className={`${s.errorContend}  ${
+            i === 0 && key === 'error' ? s.first : ''
+          }`}
+        >
+          <div className={s.verticalLine}></div>
+          <div className={s.cluster}></div>
+          <div className={s.hederError}>
+            {key === 'error' ? <Error /> : <Warning />}
+            <span className={s.infoAll}>{el}</span>
+          </div>
+        </div>
+      )
+    })
+
+    return message
+  }
+
+  const renderSkillAssistant = (
+    skill: ICustomSkill,
+    bot: BotInfoInterface,
+    i: number
+  ) => {
+    const errorCount = skill.data.error.length
+    const warningCount = skill.data.warning.length
+
+    if (errorCount === 0 && warningCount === 0) {
+      return null
+    }
+
+    const handleEditBtnClick = (e: React.MouseEvent, skill: IStackElement) => {
+      skillEditorOpened('skill_block', skill)
+      if (skill.component_type === ('Generative' as any)) {
+        nav(
+          generatePath(RoutesList.editor.skillEditor, {
+            name: bot.name as string,
+            skillId: skill.component_id,
+          } as any)
+        )
+        e.stopPropagation()
+        return
+      }
+    }
+
+    const countError = skill.data.error.length + skill.data.warning.length
+    const messageCountError = countError === 0 ? '' : `(${countError})`
+
+    const colorError =
+      skill.data.error.length !== 0
+        ? { color: '#b20000' }
+        : skill.data.warning.length !== 0
+        ? { color: '#FF9500' }
+        : {}
+
+    return (
+      <div className={s.skill} key={i}>
+        <div className={s.skillBlock}>
+          <div
+            className={s.skillName}
+            onClick={e => handleEditBtnClick(e, skill.skill)}
+          >
+            <div className={`${s.messageCountError}`} style={colorError}>
+              {messageCountError}
```

Comment:
```suggestion
    const color = skill.data.error.length
      ? 'red'
      : skill.data.warning.length
      ? 'yellow'
      : ''

    const cx = classNames.bind(s)

    return (
      <div className={s.skill} key={i}>
        <div className={s.skillBlock}>
          <div
            className={s.skillName}
            onClick={e => handleEditBtnClick(e, skill.skill)}
          >
            <div className={cx(s.messageCountError, color)}>
```
and move styles to scss